airbyte-integrations/connectors/source-mailjet-sms/source_mailjet_sms/mailjet_sms.yaml

```diff
@@ -0,0 +1,55 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: ["Data"]
+  requester:
+    url_base: "https://api.mailjet.com/v4"
+    http_method: "GET"
+    authenticator:
+      type: BearerAuthenticator
+      api_token: "{{ config['token'] }}"
+  offset_paginator:
+    type: DefaultPaginator
+    $options:
+      url_base: "*ref(definitions.requester.url_base)"
+    pagination_strategy:
+      type: "OffsetIncrement"
+      page_size: 10
```

Comment:
The limit is 100. Please use always the limit when possible. 
https://dev.mailjet.com/sms/guides/filtering-resources/#the-limit-filter